lib/active_admin/reform/engine.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails/engine'
+
+#
+module ActiveAdmin
+  #
+  module Reform
+    # Need engine to be absolutely sure active admin is already loaded, and run all it's hooks
+    class Engine < Rails::Engine
+      initializer 'activeadmin-reform' do
+        require 'active_admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.